.github/workflows/windows_ci_action.yml

```diff
@@ -0,0 +1,15 @@
+# This config builds the project on windows.
+
+name: Windows-CI
+
+# This determines when this workflow is run
+on: [push, pull_request] # on all pushes and PRs
+
+jobs:
+  CI_windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - run: mkdir build; cd build; cmake ..; cmake --build . --config Release
```

Comment:
You can simplify this folder creation with: `cmake -B build -DCMAKE_BUILD_TYPE=Release && cmake --build build`